sklearn/multioutput.py

```diff
@@ -44,37 +59,44 @@ class MultiOutputRegressor(BaseEstimator, RegressorMixin, MetaEstimatorMixin):
     estimator : estimator object
         An estimator object implementing `fit` and `predict`.
 
-    n_jobs : int, optional, default: 1
+    n_jobs : int, optional, default=1
         The number of jobs to run in parallel for `fit`. If -1,
         then the number of jobs is set to the number of cores.
         When individual estimators are fast to train or predict
         using `n_jobs>1` can result in slower performance due
         to the overhead of spawning processes.
+
+    Attributes
+    ----------
+    estimators_ : list of `n_output` estimators
+        Estimators used for predictions.
```

Comment:
You can remove the documentation here. You are rewriting it anyway in the subclasses (and no-one is going to look at this)